### PR TITLE
add App distribution from adhoc to enterprise

### PR DESCRIPTION
**Pull request by soucolline**
Added App distribution from adhoc to enterprise notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2215,6 +2215,74 @@ https://developer.apple.com/wwdc19/410
...
+- **Custom App**
+  - Part of the Apple Developer Program
+  - Privately offer customized apps to customers you identify for yourself
+  - Apps for partners, clients, franchisees, internal employees and affiliates
+  - Distribute licenses via MDM or Redemption codes
+- **Benefits of Custom Apps**
+  - One program to manage all of your apps (internal and external)
+  - Apps don’t expire
+  - Apps are managed individually
+  - Apps can be distributed to much larger audiences
```
Comment (Blackjacx) [inline README.md:2273]: Did they say how many? This would be a cool addition.

Comment (soucolline) [reply README.md:2273]: They only mentioned that unlike In-House where you could only distribute an app to employees, Custom Apps can be distributed to a broader audience such as business affiliates

Comment (Blackjacx) [reply README.md:2273]: Ok thanks 😊

**Comment by Blackjacx** [comment]
Hey there 👋

WWDC20 is knocks on your door. I thought it is a good time to ask if you liked the idea with the session notes repo and if you want it back. Since you even contributed, I would like to use the chance and invite you to give feedback to [my tweet on Twitter](https://twitter.com/Blackjacxxx/status/1273534335904362496). 

By giving feedback and/or retweeting you help to make this concept even more popular and maybe attract Apple's attention so they provide it by theirselves in future 🙃

Thank you so much in advance ✌️

**Pull request merged**